bigtable/client/row_reader.h

```diff
@@ -121,6 +124,7 @@ class RowReader {
   Filter filter_;
   std::unique_ptr<RPCRetryPolicy> retry_policy_;
   std::unique_ptr<RPCBackoffPolicy> backoff_policy_;
+  MetadataUpdatePolicy const& metadata_update_policy_;
```

Comment:
Let's just do a copy here.  I prefer to keep the memory management and lifecycles simple unless there is a compelling reason to make it more complicated.

Reply:
made the change, need to add move constructor in MetadataUpdatePolicy